src/TimeService.Validation/WorkTime/CreateWorkTimeRequestValidator.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using FluentValidation;
+using LT.DigitalOffice.Kernel.BrokerSupport.Broker;
+using LT.DigitalOffice.Models.Broker.Common;
+using LT.DigitalOffice.TimeService.Data.Interfaces;
+using LT.DigitalOffice.TimeService.Models.Dto.Requests;
+using LT.DigitalOffice.TimeService.Validation.LeaveTime;
+using LT.DigitalOffice.TimeService.Validation.WorkTime.Interfaces;
+using MassTransit;
+using Microsoft.Extensions.Logging;
+
+namespace LT.DigitalOffice.TimeService.Validation.WorkTime
+{
+  public class CreateWorkTimeRequestValidator : AbstractValidator<CreateWorkTimeRequest>, ICreateWorkTimeRequestValidator
+  {
+    private readonly IRequestClient<ICheckUsersExistence> _rcCheckUsersExistence;
+    private readonly ILogger<CreateLeaveTimeRequestValidator> _logger;
+    private readonly IWorkTimeRepository _workTimeRepository;
+
+    private async Task<bool> CheckUserExistenceAsync(List<Guid> userIds)
+    {
+      try
+      {
+        IOperationResult<ICheckUsersExistence> response = (await _rcCheckUsersExistence.GetResponse<IOperationResult<ICheckUsersExistence>>(
+          ICheckUsersExistence.CreateObj(userIds))).Message;
+
+        if (response.IsSuccess && response.Body.UserIds.Count == 1 && response.Body.UserIds[0] == userIds[0])
```

Comment:
you do not need do it response.Body.UserIds[0] == userIds[0], if user do not exist it will be empty list